modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -969,18 +971,28 @@ public void stop() {
      */
     private void cleanUpTablesResources(Map<UUID, TableImpl> tables) {
         for (TableImpl table : tables.values()) {
-            try {
-                for (int p = 0; p < table.internalTable().partitions(); p++) {
-                    TablePartitionId replicationGroupId = new TablePartitionId(table.tableId(), p);
+            for (int p = 0; p < table.internalTable().partitions(); p++) {
+                TablePartitionId replicationGroupId = new TablePartitionId(table.tableId(), p);
 
+                try {
                     raftMgr.stopRaftGroup(replicationGroupId);
+                } catch (Exception e) {
+                    LOG.info("Unable to stop the raft group [tableName={}, partition={}]", e, table.name(), p);
+                }
 
+                try {
                     replicaMgr.stopReplica(replicationGroupId);
+                } catch (Exception e) {
+                    LOG.info("Unable to stop the replica [tableName={}, partition={}]", e, table.name(), p);
                 }
+            }
 
-                table.internalTable().storage().stop();
-                table.internalTable().txStateStorage().stop();
-                table.internalTable().close();
+            try {
+                IgniteUtils.closeAll(
+                        Stream.of(table.internalTable().storage(),
```

Comment:
It's better to skip stream part. Just use
```
                IgniteUtils.closeAll(
                        table.internalTable().storage(),
                        table.internalTable().txStateStorage(),
                        table.internalTable()
                );
```

Reply:
Fixed.